src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -444,6 +453,31 @@ public ScanResult scan(ScanRequest scanRequest) {
         return scanResult;
     }
 
+    private ScanResult multiTenantScan(ScanRequest scanRequest) {
```

Comment:
Maybe `crossTenantScan` or `scanAll` (I also struggled to come up with a good name; see https://github.com/salesforce/mt-dynamo/blob/master/src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreamsBase.java#L202).

Reply:
How about `scanAllTenants`?